src/registrar/utility/csv_export.py

```diff
@@ -133,32 +124,70 @@ def write_body(
         else:
             logger.warning("csv_export -> Domain was none for PublicContact")
 
-    # all_user_nums = 0
-    # for domain_info in all_domain_infos:
-    #     user_num = len(domain_info.domain.permissions)
-    #     all_user_nums.append(user_num)
+    return security_emails_dict
+
+
+def update_columns_with_domain_managers(columns, max_dm_count):
+    """
+    Update the columns list to include "Domain manager email {#}" headers
+    based on the maximum domain manager count.
+    """
+    for i in range(1, max_dm_count + 1):
```

Comment:
This is for setting the column name and bc we are indexing at 1 instead of 0, I have to +1! 

(ie range(3) only prints out 0, 1, 2)

Reply:
Just to clarify because maybe I'm missing something:

1. The range should start at 1, but it shouldn't end in 0 
2. This should only run for the `export_data_type_to_csv` script, which means that in theory should have at least 1 domain manager -- are you mentioning the info in the case that we have 0 domain managers? ie range (1, 0 +1) where max_dm_count = 0?

Reply:
1 + 2: Ah, gotcha. I was thinking about the edge case of range(1, 0)... But I forgot that range will still return an empty array in that scenario. I was mentally mixing languages lol